tests/distributed/test_mp_dataloader.py

```diff
@@ -1,19 +1,24 @@
-import dgl
```

Comment:
I refined/formatted this file, pls pay attention to the newly added `test_multiple_dist_dataloaders` which is at the end of file. this should be fine.